.golangci.yml

```diff
@@ -32,7 +32,7 @@ linters-settings:
       - ruleguard
     settings:
       ruleguard:
-        rules: "tools/ci/rules.go"
+        rules: "./tools/ci/rules.go"
```

Comment:
I believe this was not working without the `"./"` (once I added it, then it started to detect some of the rule checks we added)